test/models/payment_history/api_importer_test.rb

```diff
@@ -0,0 +1,62 @@
+require "test_helper"
+
+class PaymentHistory::ApiImporterTest < ActiveSupport::TestCase
+  setup do
+    @user = users(:regular)
+  end
+
+  test "new" do
+    importer = PaymentHistory::ApiImporter.new(user: @user)
+
+    assert importer.user == @user
+  end
+
+  test "import!" do
+    importer = PaymentHistory::ApiImporter.new(user: @user)
+
+    ShopifyPartnerAPI.client.expects(:query).with(
+      Graphql::TransactionsQuery,
+      variables: {
+        createdAtMin: @user.calculate_from_date.iso8601,
+        cursor: ""},
```

Comment:
**[standard]** <Layout/MultilineHashBraceLayout> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.
```suggestion
        cursor: ""
},
```